Notify/Notify/Notify/ViewModels/BluetoothSettingsPageViewModel.cs

```diff
@@ -67,5 +83,65 @@ private async void onUpdateBluetoothSettingsClicked()
                 }
             }
         }
+        
+        #region Remove_Destination
+        
+        public Command RemoveBluetoothDestinationCommand { get; set; }
+        
+        private async void onRemoveBluetoothDestinationClicked()
+        {
+            bool successfulUpdate;
+            bool isConfirmed = await App.Current.MainPage.DisplayAlert("Confirmation", $"Are you sure you want to remove {SelectedLocation} bluetooth destination from preferences?", "Yes", "No");
+
+            if (isConfirmed)
+            {
+                successfulUpdate = AzureHttpClient.Instance.RemoveDestination(m_SelectedLocation, NotificationType.Bluetooth).Result;
+                
+                if (successfulUpdate)
+                {
+                    App.Current.MainPage.DisplayAlert("Remove", $"Remove success!", "OK");
+                    await AzureHttpClient.Instance.GetDestinations();
+                }
+                else
+                {
+                    App.Current.MainPage.DisplayAlert("Error", "Something went wrong", "OK");
+                }
+            }
+        }
+        
+        private string m_RemoveBluetoothButtonText = "PLEASE CHOOSE DESTINATION";
+        public string RemoveBluetoothButtonText
+        {
+            get => m_RemoveBluetoothButtonText;
+            set => SetField(ref m_RemoveBluetoothButtonText, value);
+        }
```

Comment:
Is the text in the button should be changed? 
If so, to what text?

Reply:
Have you tried to run the application?